application/src/test/java/org/togetherjava/tjbot/jda/JdaTester.java

```diff
@@ -487,6 +495,110 @@ public JdaTester() {
+    }
+
+    /**
+     * Creates a message spy
+     *
+     * @return the created spy
+     */
+    public @NotNull Message createMessageSpy() {
+        return spy(Message.class);
+    }
+
+    /**
+     * Creates an attachment
+     *
+     * @param id id of the attachment
+     * @param url url of the attachment
+     * @param proxyUrl proxyUrl of the attachment
+     * @param fileName fileName of the attachment
+     * @param contentType contentType of the attachment
+     * @param description description of the attachment
+     * @param size size of the attachment
+     * @param height height of the attachment
+     * @param width width of the attachment
+     * @param ephemeral is the attachment ephemeral
+     * @return the created attachment
+     */
+    public Message.Attachment createAttachment(long id, String url, String proxyUrl,
+            String fileName, String contentType, String description, int size, int height,
+            int width, boolean ephemeral) {
+        return new Message.Attachment(id, url, proxyUrl, fileName, contentType, description, size,
+                height, width, ephemeral, jda);
+    }
+
+    /**
+     * Mocks the attachments from the given message
+     *
+     * @param message the message with the mocked attachments
+     * @param attachments the attachments which the mock should return
+     */
+    public void mockMessageAttachments(Message message, List<Message.Attachment> attachments) {
+        when(message.getAttachments()).thenReturn(attachments);
+    }
+
+    /**
+     * Creates an embedded message
+     *
+     * @param url url of the embedded message
+     * @param title title of the embedded message
+     * @param description description of the embedded message
+     * @param type type of the embedded message
+     * @param timestamp timestamp of the embedded message
+     * @param color color of the embedded message
+     * @param thumbnail thumbnail of the embedded message
+     * @param siteProvider siteProvider of the embedded message
+     * @param author author of the embedded message
+     * @param videoInfo videoInfo of the embedded message
+     * @param footer footer of the embedded message
+     * @param image image of the embedded message
+     * @param fields fields of the embedded message
+     * @return the created embedded message
+     */
+    public MessageEmbed createMessageEmbed(String url, String title, String description,
+            EmbedType type, OffsetDateTime timestamp, int color, MessageEmbed.Thumbnail thumbnail,
+            MessageEmbed.Provider siteProvider, MessageEmbed.AuthorInfo author,
+            MessageEmbed.VideoInfo videoInfo, MessageEmbed.Footer footer,
+            MessageEmbed.ImageInfo image, List<MessageEmbed.Field> fields) {
+        return new MessageEmbed(url, title, description, type, timestamp, color, thumbnail,
+                siteProvider, author, videoInfo, footer, image, fields);
+    }
```

Comment:
Same complaint with the forward instead of helper here as well. There is already `EmbedBuilder` which should be used instead.